ui/packages/consul-ui/app/components/topology-metrics/card/index.hbs

```diff
@@ -29,18 +29,35 @@
     <TopologyMetrics::SourceType @source='routing-config' />
   {{/if}}
   <div class="details">
-  {{#if (and (env 'CONSUL_NSPACES_ENABLED') (not-eq @item.Namespace @service.Namespace))}}
-    <dl class="nspace">
-      <dt>
-        <Tooltip>
-            Namespace
-        </Tooltip>
-      </dt>
-      <dd>
-        {{@item.Namespace}}
-      </dd>
-    </dl>
-  {{/if}}
+    {{#if (and (can 'use partitions') (can 'use nspaces'))}}
+    <div class={{if (not-eq @item.Partition @service.Partition) 'group'}}>
+      {{#if (not-eq @item.Partition @service.Partition)}}
+      <dl class="partition">
+        <dt>
+          <Tooltip>
+              Partition
```

Comment:
P.S. Just realised in Bucket::List the tooltip reads 'Admin Partition', tbh not sure which one we should use really, but they should be consistent, so if you could change this one up to Admin Partition while you are here that'd be great!

Reply:
Updated to say `Admin Partition`